selfdrive/controls/lib/lateral_planner.py

```diff
@@ -47,9 +47,13 @@
 
 
 class LateralPlanner():
-  def __init__(self, CP):
+  def __init__(self, CP, use_lanelines=None):
```

Comment:
Should we just pass in both toggle values and have later_planner read the params?

Reply:
Ah yes good idea